### PR TITLE
https://issues.redhat.com/browse/ACM-10554 Add min ISO troubleshoot

### PR DESCRIPTION
**Pull request by oafischer**
2.9 and possibly 2.10 and 2.8 as well.

### CODE REVIEW

File: clusters/support_troubleshooting/troubleshooting_mce_intro.adoc [hunk 1]
```diff
@@ -37,4 +37,5 @@ To view the main documentation about managing your clusters, see xref:../cluster
  * xref:../support_troubleshooting/trouble_cluster_remove_namespace_mce.adoc#trouble-cluster-remove-namespace-mce[Namespace remains after deleting a cluster]
  * xref:../support_troubleshooting/trouble_auto_import_secret_exists_mce.adoc#trouble-auto-import-secret-exists-mce[Auto-import-secret-exists error when importing a cluster]
  * xref:../support_troubleshooting/trouble_no_placementdecision_mce.adoc#troubleshooting-no-placementdecision-mce[Troubleshooting missing PlacementDecision after creating Placement]
-* xref:../support_troubleshooting/trouble_idrac_discovery_fails_mce.adoc#troubleshooting-idrac-discovery-fails-mce[Troubleshooting a discovery failure of bare metal hosts on Dell hardware]
+ * xref:../support_troubleshooting/trouble_idrac_discovery_fails_mce.adoc#troubleshooting-idrac-discovery-fails-mce[Troubleshooting a discovery failure of bare metal hosts on Dell hardware]
+ * xref:../support_troubleshooting/trouble_min_iso_boot.adoc#trouble-min-iso-boot[Minimal ISO boot failures]
```
Comment (dockerymick) [inline clusters/support_troubleshooting/troubleshooting_mce_intro.adoc]: I know that your changes are not related, but there is a broken link in this topic:
Screenshot 2024-04-12 at 7 51 21 AM

File: clusters/support_troubleshooting/trouble_min_iso_boot.adoc [hunk 2]
```diff
@@ -0,0 +1,14 @@
+[#trouble-min-iso-boot]
+= Minimal ISO boot failures
```
Comment (dockerymick) [inline clusters/support_troubleshooting/trouble_min_iso_boot.adoc]:
```suggestion
= Troubleshooting Minimal ISO boot failures
```

File: clusters/support_troubleshooting/troubleshooting_mce_intro.adoc (hunk 1 above)
Comment (dockerymick) [inline clusters/support_troubleshooting/troubleshooting_mce_intro.adoc]:
```suggestion
 * xref:../support_troubleshooting/trouble_min_iso_boot.adoc#trouble-min-iso-boot[Troubleshooting Minimal ISO boot failures]
```

**Review by dockerymick — APPROVED** [review]
There is a broken link that is not related to the changes that you committed. Will you fix the link? Besides the suggestions for the title and link, it looks good to me! Thanks Oliver

**Review by jgato — APPROVED** [review]

**Review by jgato — APPROVED** [review]

**Review by dockerymick — APPROVED** [review]

**Pull request merged**